addons/sale/views/sale_portal_templates.xml

```diff
@@ -19,18 +19,28 @@
     </template>
 
     <template id="portal_my_home_sale" name="Show Quotations / Sales Orders" customize_show="True" inherit_id="portal.portal_my_home" priority="20">
-        <xpath expr="//div[hasclass('o_portal_docs')]" position="inside">
+        <xpath expr="//div[hasclass('o_portal_docs')]" position="before">
+            <t t-set="portal_client_category_enable" t-value="True"/>
+            <t t-set="portal_alert_category_enable" t-value="True"/>
+        </xpath>
+        <div id="portal_alert_category" position="inside">
             <t t-call="portal.portal_docs_entry">
-                <t t-set="title">Quotations</t>
-                <t t-set="url" t-value="'/my/quotes'"/>
+                <t t-set="bg_color" t-value="'alert alert-primary'"/>
                 <t t-set="placeholder_count" t-value="'quotation_count'"/>
+                <t t-set="title">Quotation(s) to review</t>
```

Comment:
```suggestion
                <t t-set="title">Quotations to review</t>
```

Reply:
Makes sense to me, addressed here [e114976](https://github.com/odoo/odoo/pull/138077/commits/e1149768d889bd3d668884d574a51221052bce9a) 👍